permissions/plugin-GatekeeperPlugin.yml

```diff
@@ -5,4 +5,5 @@ paths:
 - "org/paylogic/GatekeeperPlugin"
 developers:
 - "bubenkoff"
-- "maikelwever"
\ No newline at end of file
+- "maikelwever"
+issues: "github:jenkinsci/gatekeeper-plugin"
```

Comment:
do we need to specify the github repo for the issue tracker or should we be pulling that from the `github` field? as a backup we could have just github and allow overriding it.

I'm concerned we're duplicating information when we don't need to

Reply:
> I'm concerned we're duplicating information when we don't need to

I never want to guess. 

I also don't see the problem with duplicating the information, is there a byte limit thing your concerned about?

Reply:
it's not guessing though, a github issue tracker is tied to the repo.

I'm not aware of any plugin that tracks issues in another github issue tracker (not to say we can't support it)

> I also don't see the problem with duplicating the information, is there a byte limit thing your concerned about?

no it's just people having to duplicate information, why do it?